src/gps_to_pose_conversion_node.cpp

```diff
@@ -76,6 +77,13 @@ void gps_callback(const sensor_msgs::NavSatFixConstPtr& msg)
   pose_msg->pose.pose.position.z = z;
   pose_msg->pose.pose.orientation = g_latest_imu_msg.orientation;
 
+  // Fill up position message
+  geometry_msgs::PointStampedPtr position_msg(
+    new geometry_msgs::PointStamped);
+  position_msg->header = pose_msg->header;
+  position_msg->header.frame_id = "world";
+  position_msg->point = pose_msg->pose.pose.position;
+
   if (g_got_pressure) {
```

Comment:
Please add the option of taking z from laser_altimeter, with a boolean parameter allowing to switch between gps altitude and laser altitude. Just add a subscriber that allows to take a Float64 as "external_altitude" input, keeping it general (as opposed to mentioning the laser altimeter).